README.md

```diff
@@ -1,5 +1,35 @@
 # Readium Desktop
 
+Import and read EPUB ebooks on your Windows 10/10S computer. Great reading experience; local bookshelf with search capabilities and personal
```

Comment:
"on your Windows 10/10S computer"
=>
I think that we should mention MacOS and Linux here, otherwise people will think Thorium is Windows-only.